projects/components/src/quick-search/quick-search.provider.ts

```diff
@@ -72,6 +72,11 @@ export interface QuickSearchProvider {
      * Says whether this provider can return results with the given filter applied.
      */
     canHandleFilter?(filter: ActiveQuickSearchFilter): boolean;
+
+    /**
+     * The icon that is displayed next to the search section title.
```

Comment:
It will be good to mention that this icon is the `shape` property as defined by clarity